conf/radiusd/eap.conf.example

```diff
@@ -268,8 +268,29 @@ eap [% key -%] {
         # the EAP keys correctly.  The fix is to upgrade
         # OpenSSL, or disable TLS 1.2 here. 
 #       disable_tlsv1_2 = no
+        disable_tlsv1_1 = yes
+        disable_tlsv1 = yes
+        #
 
+        #  Set min / max TLS version.  Mainly for Debian
+        #  "trusty", which disables older versions of TLS, and
+        #  requires the application to manually enable them.
+        #
+        #  If you are running Debian trusty, you should set
+        #  these options, otherwise older clients will not be
+        #  able to connect.
+        #
+        #  Allowed values are "1.0", "1.1", "1.2", and "1.3".
+        #
+        #  Note that the server WILL NOT permit negotiation of
+        #  TLS 1.3.  The EAP-TLS standards for TLS 1.3 are NOT
+        #  finished.  It is therefore impossible for the server
+        #  to negotiate EAP-TLS correctly with TLS 1.3.
+        #
+        #  The values must be in quotes.
         #
+        tls_min_version = "1.2"
+        tls_max_version = "1.2"
```

Comment:
Should this be configurable?

Reply:
Yes, I commented above regarding that, this should be allowed to be configured in the admin GUI and we should document the behavior of a client that doesn't support 1.2 so that we know that we need to change this value